.github/workflows/sanity-test.yaml

```diff
@@ -29,14 +29,18 @@ jobs:
       AZURE_CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
       AZURE_TENANT_ID: ${{ secrets.TENANT_ID }}
       CORSO_BUCKET: ${{ secrets.CI_TESTS_S3_BUCKET }}
-      CORSO_LOG_FILE: ./src/testlog/testlogging.log
+      CORSO_LOG_DIR: testlog
+      CORSO_LOG_FILE: ${{ env.CORSO_LOG_DIR }}/testlogging.log
```

Comment:
Does the log file get cleaned up post run? I've noticed an interesting behavior with corso - If you supply an existing log file to corso, it concatenates to it rather than overwriting it. 
Is this by design? If the testlogging.log file doesn't get cleaned up, it might get really big across runs. Also, this we might not want this.

Reply:
the log file won't be cleaned up between runs

I didn't think about log file size, mostly just how to make it more readable by adding some extra output in there for the different tests. I was looking to see if there was a way to extract the name of the current step that's running, but unfortunately that doesn't appear possible. If it was that would make for an easy way to set the log name

I can split the log into different files if that would be useful, otherwise we can test this out and adjust as needed

Reply:
hmmm actually may be able to dig more for how to extract the step name 🤔

Reply:
ah, no it has no good way to do that. Only accessing by ID can you get the name, but there's no way to get the ID of the current step it seems

Reply:
Yeah, your changes are fine. I was just commenting on the evergrowing corso log file. 
We should be able to verify the _evergrowingness_ from archives. Also, we could switch to running jobs in a container, so that state gets cleaned up after runs. But I don't know if this has any other repercussions. 

https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container